pkg/storage/sqlite/sqlite_test.go

```diff
@@ -0,0 +1,358 @@
+	)
+	require.NoError(t, err)
+
+	tk := tuple.NewTupleKey("folder:2021-budget", "owner", "user:anne")
+	iter, err := ds.Read(ctx, "store", tk, storage.ReadOptions{})
+	require.NoError(t, err)
+	defer iter.Stop()
+
+	curTuple, err := iter.Next(ctx)
+	require.NoError(t, err)
+	require.Equal(t, tk, curTuple.GetKey())
+
+	opts := storage.ReadPageOptions{
+		Pagination: storage.NewPaginationOptions(2, ""),
+	}
+	tuples, _, err := ds.ReadPage(ctx, "store", &openfgav1.TupleKey{}, opts)
+	require.NoError(t, err)
+	require.Len(t, tuples, 1)
+	require.Equal(t, tk, tuples[0].GetKey())
+
+	userTuple, err := ds.ReadUserTuple(ctx, "store", tk, storage.ReadUserTupleOptions{})
+	require.NoError(t, err)
+	require.Equal(t, tk, userTuple.GetKey())
+
+	tk2 := tuple.NewTupleKey("folder:2022-budget", "viewer", "user:anne")
+	_, err = ds.db.ExecContext(
+		ctx, stmt, "store", "folder", "2022-budget", "viewer", "user:anne", "userset",
+		ulid.Make().String(), nil, nil,
+	)
+
+	require.NoError(t, err)
+	iter, err = ds.ReadUsersetTuples(ctx, "store", storage.ReadUsersetTuplesFilter{Object: "folder:2022-budget"}, storage.ReadUsersetTuplesOptions{})
+	require.NoError(t, err)
+	defer iter.Stop()
+
+	curTuple, err = iter.Next(ctx)
+	require.NoError(t, err)
+	require.Equal(t, tk2, curTuple.GetKey())
+
+	iter, err = ds.ReadStartingWithUser(ctx, "store", storage.ReadStartingWithUserFilter{
+		ObjectType: "folder",
+		Relation:   "owner",
+		UserFilter: []*openfgav1.ObjectRelation{
+			{Object: "user:anne"},
+		},
+	}, storage.ReadStartingWithUserOptions{})
+	require.NoError(t, err)
+	defer iter.Stop()
+
+	curTuple, err = iter.Next(ctx)
+	require.NoError(t, err)
+	require.Equal(t, tk, curTuple.GetKey())
+
+	stmt = `
+	INSERT INTO changelog (
+		store, object_type, object_id, relation, _user, ulid,
+		condition_name, condition_context, inserted_at, operation
+	) VALUES (?, ?, ?, ?, ?, ?, ?, ?, datetime('subsec'), ?);
+`
+	_, err = ds.db.ExecContext(
+		ctx, stmt, "store", "folder", "2021-budget", "owner", "user:anne",
+		ulid.Make().String(), nil, nil, openfgav1.TupleOperation_TUPLE_OPERATION_WRITE,
+	)
+	require.NoError(t, err)
+
+	_, err = ds.db.ExecContext(
+		ctx, stmt, "store", "folder", "2021-budget", "owner", "user:anne",
+		ulid.Make().String(), nil, nil, openfgav1.TupleOperation_TUPLE_OPERATION_DELETE,
+	)
+	require.NoError(t, err)
+
+	readChangesOpts := storage.ReadChangesOptions{
+		Pagination: storage.NewPaginationOptions(storage.DefaultPageSize, ""),
+	}
+	changes, _, err := ds.ReadChanges(ctx, "store", "folder", readChangesOpts, 0)
+	require.NoError(t, err)
+	require.Len(t, changes, 2)
+	require.Equal(t, tk, changes[0].GetTupleKey())
+	require.Equal(t, tk, changes[1].GetTupleKey())
+}
+
+// TestMarshalledAssertions tests that previously persisted marshalled
+// assertions can be read back. In any case where the Assertions proto model
+// needs to change, we'll likely need to introduce a series of data migrations.
+func TestMarshalledAssertions(t *testing.T) {
```

Comment:
We don't need this test, it was only applicable to MySQL and Postgres https://github.com/openfga/openfga/pull/1217

Reply:
Cool, I'm all for keeping things clean 😄  Intent here was mostly to demonstrate that this backend has parity with mysql and postgres, and can pass all the same tests.